api/api/versions_api_test.go

```diff
@@ -511,3 +513,137 @@ func TestPatchVersion(t *testing.T) {
 		})
 	}
 }
+
+type mockClient struct{}
```

Comment:
With the current project structure, I think it's cleaner if this is defined in a different package, e.g `mlflow.MockClient`, `mocks.MLflowClient` or smth similar.

Currently it's defined in `api` package and `api.mockClient` can be quite misleading when this is actually a fake / mock **MLflow** client. If we want to keep it here, at least we need to maybe prefix the name like `mockMLflowClient`

Reply:
alright.